src/constants.ts

```diff
@@ -46,4 +46,5 @@ export const MULTIDISCIPLINARY_SUBJECT_ARTICLE_PAGE_PATH =
   '/subject\\::subjectId(d1fe9d0a-a54d-49db-a4c2-fd5463a7c9e7)/topic\\::topic1/topic\\::topic2/:topicId';
 
 export const RESOURCE_ARTICLE_IFRAME_PATH = `/article-iframe/:lang?/urn\\:resource\\::resourceId/:articleId`;
+export const TOPIC_ARTICLE_IFRAME_PATH = `/article-iframe/:lang?/urn\\::topicId/:articleId`;
```

Comment:
Så lenge du ikkje har definert denne `urn\\::topic\\::topicId` så er du avhengig av at den over matches mot først. Dersom nokon skule endre rekkefølgen i urlHelper for å feks sortere så vil jo denne matche mot både topic og ressurs.

Reply:
Yep, samme med `RESOURCE_PAGE_PATH` og `TOPIC_PATH`.
Om jeg endrer den så må jeg også endre de andre for å hente ut "like mye" av variabelen (topic:1:1234 vs 1:1234).
Eller ha en eller annen sjekk for å legge det på.

Funker på samme måte med de vanlige rutene i `routes.tsx`.

Reply:
TOPIC_PATH kunne vi egentlig fjerna da den ikkje er i bruk.